script/dev-server

```diff
@@ -3,6 +3,11 @@ set -euo pipefail
 IFS=$'\n\t'
 PATH=`npm bin`:$PATH
 
+if [ -z "${ISS_URL:=}" ] ; then
+    echo "ISS_URL not set; cannot run."
+    exit 1
+fi
```

Comment:
Nodemon doesn't quit if the process crashes instantly up-front, so then you have to ctrl-c `nodemon`.

Also, the output gets surrounded by a bunch of colorized nodemon stuff which is hard to read, and the focus is stolen by the browser window opening.